src/pkg/message/message.go

```diff
@@ -340,3 +340,25 @@ func debugPrinter(offset int, a ...any) {
 func errorPrinter(offset int) *pterm.PrefixPrinter {
 	return pterm.Error.WithShowLineNumber(logLevel > 2).WithLineNumberOffset(offset)
 }
+
+// Table prints a padded table containing the specified header and data
+func Table(header []string, data [][]string) {
```

Comment:
Usually when ordering the functions in a file the exported functions are above the unexported ones.  (so `Table` would go above `debugPrinter`)